android/app/src/main/java/co/nimblehq/smsforwarder/ui/screens/login/LoginViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.smsforwarder.ui.screens.login
+
+import androidx.hilt.lifecycle.ViewModelInject
+import co.nimblehq.smsforwarder.ui.base.BaseViewModel
+import co.nimblehq.smsforwarder.ui.base.NavigationEvent
+
+interface Input {
+    fun navigateToFilter()
+}
```

Comment:
🤔  `navigateToFilter` is the input? IMO, `Input` is being used for passing value from Fragment to VM, in this case, I think we could create `inteface LoginViewModel` and current `class LoginViewModel` will be implemented the Interface. Ex:

```
interface LoginViewModel {
    fun navigateToFilter()
}

class LoginViewModel @ViewModelInject constructor() : BaseViewModel(), LoginViewModel {
}
```

How do you think?

Reply:
So if this viewModel has input and output also, it will have 3 interfaces, right?

Reply:
Yup 😄

Reply:
Then I have a question, If some page we have only input and output, it will not consistent. What do you think about this case?

Reply:
Just ignore Input and Output if no usage 😄  But the interface for ViewModel is required 😄